TweetMate/build.gradle

```diff
@@ -0,0 +1,77 @@
+apply plugin: 'com.android.application'
+
+def file = rootProject.file("local.properties");
+def prop = new Properties()
+prop.load(new FileInputStream(file))
+
+android {
+    compileSdkVersion 31
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        applicationId "com.ybsystem.tweethub"
+        minSdkVersion 26
+        targetSdkVersion 31
+        versionCode 20
+        versionName "2.0.3"
+        buildConfigField("String", "CK", prop['ck'])
+        buildConfigField("String", "CS", prop['cs'])
+        buildConfigField("String", "CB", prop['cb'])
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    lintOptions {
+        disable 'SpUsage'
+        disable 'SetTextI18n'
+        disable 'IconLocation'
+        disable 'HardcodedText'
+        disable 'UnusedResources'
+        disable 'ContentDescription'
+        disable 'NonConstantResourceId'
+        disable 'AnnotationProcessorOnCompilePath'
+    }
+}
+
+dependencies {
+    compileOnly 'org.projectlombok:lombok:1.18.12'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'androidx.preference:preference:1.1.1'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation "androidx.coordinatorlayout:coordinatorlayout:1.1.0"
+    implementation 'com.google.android.material:material:1.3.0'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetMate-for-Android/blob/0a5013e538213bacb2774f24bfa4ed6959fa720c/TweetMate/build.gradle#L58"></span>A newer version of com.google.android.material:material than 1.3.0 is available: 1.5.0</td>
    </tr>
  </tbody>
</table>